dpdk/vr_dpdk_knidev.c

```diff
@@ -199,6 +199,27 @@ struct rte_port_out_ops dpdk_knidev_writer_ops = {
     .f_flush = dpdk_knidev_writer_flush,
 };
 
+/* Release KNI RX queue */
+static void
+dpdk_kni_rx_queue_release(unsigned lcore_id, struct vr_interface *vif)
```

Comment:
Will this be called on lcores which do not have a RX queue assigned to them (as vhost0 has only one rx queue)?